app/Http/Controllers/SchoolsController.php

```diff
@@ -59,7 +63,28 @@ public function destroy(School $school): RedirectResponse
         $school->delete();
 
         return redirect()
-            ->back();
+            ->back()
+            ->with("status", "Szkoła została usunięta.");
+    }
+
+    public function disable(School $school): RedirectResponse
+    {
+        $school->is_disabled = true;
+        $school->save();
+
+        return redirect()
+            ->back()
+            ->with("status", "Szkoła została zablokowa.");
+    }
+
+    public function enable(School $school): RedirectResponse
+    {
+        $school->is_disabled = false;
+        $school->save();
+
+        return redirect()
+            ->back()
+            ->with("status", "Szkoła została odblokowana.");
     }
```

Comment:
I believe that we can made one method - something like

```
public function toggleDisable(School $school, bool $disable): RedirectResponse
{
    $school->is_disabled = $disable;
    $school->save();

    $message = $disable ? "Szkoła została zablokowana." : "Szkoła została odblokowana.";

    return redirect()
        ->back()
        ->with("status", $message);
}
```

Reply:
fixed